### DISCUSSION TITLE
No options to create playlists?

### DISCUSSION THREAD

**Discussion by KyleMaas** [mediacms-io/mediacms#544] [General]
If I log in as a user and navigate to My Playlists, I see a heading of "Created playlists" but I see no way to actually create a playlist.  What am I missing?

**Comment by mgogoulos** [comment]
you may create a playlist while browsing a video, you will see the option Save under the video if you are logged in. Agree that could be added on other places too (eg on the video area itself) and i think there is a ticket for this since a long time!

**Reply by KyleMaas** [reply]
Ah, yes, that works.  Thanks!
